controllers/progressivesync_controller_test.go

```diff
@@ -91,8 +92,8 @@ var _ = Describe("ProgressiveRollout Controller", func() {
 		namespace, ns = createRandomNamespace()
```

Comment:
Could we call this `ctrlNamespace` or something to make it a bit clearer?

Reply:
Yep, done.